src/frontend/Sider/Behandling/VedtakOgBeregning/Barnetilsyn/InnvilgeVedtak/vedtaksvalidering.ts

```diff
@@ -147,6 +153,59 @@ const validerStønadsperioder = (
     };
 };
 
+const validerUtgifter = (
+    utgifter: Record<string, Utgift[]>
+): FormErrors<{
+    utgifter: Record<string, Utgift[]>;
+}> => {
+    const feilIUtgifter = Object.entries(utgifter).reduce((acc, [barnId, utgifter]) => {
+        return {
+            ...acc,
+            [barnId]: utgifter.map((utgift) => {
+                const utgiftFeil = {
+                    fra: undefined,
+                    til: undefined,
+                    utgift: undefined,
+                };
+
+                // TODO: Tilpass validering
+                if (!utgift.fra) {
+                    return { ...utgiftFeil, fra: 'Mangler fradato for periode' };
+                }
+
+                if (!utgift.til) {
+                    return { ...utgiftFeil, til: 'Mangler tildato for periode' };
+                }
+
+                if (!erDatoEtterEllerLik(utgift.til, utgift.fra)) {
+                    return {
+                        ...utgiftFeil,
+                        til: 'Sluttdato (til) må være etter startdato (fra) for periode',
+                    };
+                }
```

Comment:
Når vi returner ved første feil, gjør det da at man bare får en valideringsfeil av gangen pr. utgiftsvalg? Hvis man feks mangler å fylle ut både `fra` og `utgift`, vil man da få feil på begge med en gang. Eller bare feil på `fra`?

Reply:
Ja det stemmer. Det er sånn det er løst i EF så har ikke gjort noe endringer foreløpig. Sjekkene som ligger der er kun det jeg alt hadde lagd/kopiert forrige uke og ikke nødvendigvis det som skal sjekkes. Det måtte bare være noe der for at formstate ikke skulle gi feilmeldinger. 

Tenker å ha en egen PR på validering og før det diskutere hvordan feilmeldinger faktisk skal vises. Min tanke her er å droppe return under alle og returnere på slutten slik at man får opp alle feilmeldinger samtidig.